### PR TITLE
Feature/default profile img

### PR DESCRIPTION
**Pull request by Willcowen**
default-profile-img-assigned
Added a default profile img using a public 'avatars' api, it will generate an image based upon the first two letters of the persons name, as a placeholder.

### CODE REVIEW

**Review by nicopicchio — CHANGES_REQUESTED** [review]
It looks great on the side rather than on top. Perhaps I would make the image a bit bigger so that its height is the same as the user details. And maybe I would also give it a 50% border radius to make it round?

**Comment by Willcowen** [comment]
> 

The profile img is returned from a public api, so the img is quite small, if I make it larger it will be blurry. What do you think? I can have a go at making it round for sure.

File: src/components/users/userProfile/UserProfile.js [hunk 1]
```diff
@@ -3,33 +3,50 @@ import { useState, useEffect } from 'react';
...
+      .catch((err) => console.log(err.response));
+  };
+
   return (
     <div>
-      <Header role={role}/>
+      <Header role={role} />
       <h1>User Profile</h1>
+      <div className="profile">
+      <div className="profile-img">
```
Comment (mikemherron) [inline src/components/users/userProfile/UserProfile.js:38]: Indent your markup 😭 😜

**Review by mikemherron — APPROVED** [review]
Small change to format; agree with Nicos styling suggestions

File: src/components/users/userProfile/UserProfile.js (hunk 1 above)
Comment (Willcowen) [reply src/components/users/userProfile/UserProfile.js]: Apologies! Just formatted it again and re submitted with the round img.

**Review by nicopicchio — APPROVED** [review]
like the new styling. this might change in the future but for now is more than enough

**Review by mikemherron — APPROVED** [review]

**Pull request merged**